### ISSUE TITLE
needed to edit docker-compose to function

### ISSUE THREAD

**Issue by rossc123** [MattsTechInfo/Meshnet#16]
Hi, not sure if this could be a widespread issue or just an annomaly in my case. (using unraid btw)
When I initally set up this docker container I couldn't get any of my devices to connect.
It was only after I added
`network_mode:"host"` to docker-compose.yml that I was able to get it working properly.

Now it works great, thanks for making this!

**Comment by s116821** [comment]
Were you actually able to connect to unraid smb share or its management webpage via the nord hostname using this meshnet container @rossc123 ? I've managed to get the host to come up as a new device in the meshnet list of current online devices with both host and bridged modes, but have never been able to access the web management page or the smb share via the nord hostname.  I can ping the nord hostname just fine though.

**Comment by rossc123** [comment]
I mostly use meshnet for Jellyfin. I haven't tried accessing smb or the web management page with meshnet

**Comment by MattsBos** [comment]
Hi @rossc123 , thanks for mentioning. Different deployments will required different network settings.
I'm using this container from Kubernetes and did some local testing while building in docker, using it as an exit node for routing. That's also what I based the samples on, just a basic example to get things started and your node connected to Meshnet.

I am aware docker has some built in protection with iptables to avoid some potential security risks, so some connections might be  a bit iffy to get working.

**Comment by s116821** [comment]
Yea I ended up just going with tailscale b/c the Windows Nord desktop client app kills my internet speeds (900 Mb down and 100 up vs 3600 down and 1700 up w/out the Nord desktop app) and LAN speeds (all local tests via iperf were tremendously slow when the Nord app was running, an issue only encountered on my Windows systems).  I wanted to stay in the Nord ecosystem and make it work but am glad I made the change.  One thing I did see but never tried it out is that the Unraid smb share settings have a config option to set the listening interfaces, I wonder if adding the docker interface there would have helped with my issue and made the meshnet container work for my needs.

Anyways thanks for the responses anyways.
